packages/web-components/src/checkbox/README.md

```diff
@@ -0,0 +1,121 @@
+
+[Link to Checkbox Design Spec in Figma](https://www.figma.com/file/1a1hBVizk7aLH76IvrnPFs/Checkbox?node-id=1666-534&t=Y5ISi4tWguXRFMdP-0)
+
+<br />
+
+## **Engineering Spec**
+
+Fluent WC3 Checkbox extends from the [FAST Checkbox](https://explore.fast.design/components/fast-checkbox) and is intended to be as close to the Fluent UI React 9 Checkbox implementation as possible. However, due to the nature of web components there will not be 100% parity between the two.
+
+<br />
+
+## Class: `Checkbox`
+
+<br />
+
+### **Component Name**
+
+`<fluent-checkbox>`
+
+<br />
+
+### **Component Template**
+
+FAST `checkboxTemplate`
+
+<br />
+
+### **Variables**
+
+| Name                    | Description          | Type                                         |
+| ----------------------- | -------------------- | -------------------------------------------- |
+| `checkboxLabelPosition` | label position types | `{ before: "before", after: "after" }`       |
+| `checkboxShape`         | checkbox shape types | `{ square: "square", circular: "circular" }` |
+| `checkboxSize`          | checkbox size types  | `{ medium: "medium", large: "large" }`       |
+
+<br />
+
+### **Fields**
+
+| Name            | Privacy | Type                    | Default                       | Description                                        |
+| --------------- | ------- | ----------------------- | ----------------------------- | -------------------------------------------------- |
+| `labelPosition` | public  | `CheckboxLabelPosition` | `CheckboxLabelPosition.after` | Indicates postion of label                         |
+| `checked`       | public  | `boolean`               | `false`                       | Indicates whether input is checked                 |
+| `indeterminate` | public  | `boolean`               | `false`                       | Indicates whether input is initially indeterminate |
+| `disabled`      | public  | `boolean`               | `false`                       | Indicates whether input is disabled                |
+| `required`      | public  | `boolean`               | `false`                       | Indicates whether input is required                |
+| `size`          | public  | `CheckboxSize`          | `CheckboxSize.medium`         | Indicates the size of the checkbox                 |
+| `shape`         | public  | `CheckboxShape`         | `CheckboxShape.square`        | Indicates shape of the checkbox                    |
+
+<br />
+
+### **Methods**
+
+| Name              | Privacy | Description             |
+| ----------------- | ------- | ----------------------- |
+| `keypressHandler` | public  | fires on keyboard press |
+| `clickHandler `   | public  | fires on mouse click    |
+
+### **Events**
+
+| Name     | Type | Description                 | Inherited From |
+| -------- | ---- | --------------------------- | -------------- |
+| `change` |      | Fires a custom change event | `FASTCheckbox` |
+
+<br />
+
+### **Attributes**
```

Comment:
Do we need to document **Fields** and **Attributes** separately?

Reply:
I have been following FASTUI docs on these but this has come up a lot so I have removed the fields table and made sure everything is documented in the attributes table.